### ISSUE TITLE
Use saved Wireshark files

### ISSUE THREAD

**Issue by Arjun-Deodhar** [Arnav2Prasad/HTTP-Server#3]
Wireshark allows you to save .pcapng format files that contain all the details of the wireshark session

We can also use those for viewing the HTTP request response messages

**Comment by Arnav2Prasad** [comment]
Yes, viewed it. It would be quite useful and much needed to save all response from now onwards. We will need to analyze all packets request for our project.

We can also create a directory for storing every day packet-details which will help us store all related files at one place and will be easy for further analysis.
(While working on the code we could cross-check our code with these packets then!)

**Comment by Arjun-Deodhar** [comment]
Yes, Sure!
